content/en/news/releases/1.15.x/announcing-1.15.1/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Announcing Istio 1.15.1
+linktitle: 1.15.1
+subtitle: Patch Release
+description: Istio 1.15.1 patch release.
+publishdate: 2022-09-23
+release: 1.15.1
+---
+
+This release contains bug fixes to improve robustness.
+
+This release note describes what is different between Istio 1.15.0 and Istio 1.15.1.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** `PILOT_ENABLE_K8S_SELECT_WORKLOAD_ENTRIES` feature back to Istio which was removed in 1.14. Will
```

Comment:
Nit, and I know this isn't the RM's fault to fix: this doesn't really tell me anything about what the feature is, why it was removed, etc.  I would at least not lead with it in the change list.

Reply:
Added link to the PR where it was discussed at least for some context and moved from top.